blazor/pdfviewer/getting-started/web-assembly-application.md

```diff
@@ -163,3 +165,43 @@ Press <kbd>Ctrl</kbd>+<kbd>F5</kbd> (Windows) or <kbd>⌘</kbd>+<kbd>F5</kbd> (m
 ![Blazor PDF Viewer Component](GettingStarted_images/blazor-pdfviewer.png)
 
 > [View Sample in GitHub](https://github.com/SyncfusionExamples/Blazor-Getting-Started-Examples/tree/main/PDFViewer).
+
+## How to load desired PDF for initial loading and change document path at runtime
```

Comment:
**Load the desired PDF at initial loading and change the document path at runtime.**

Can you please modify like this?

Reply:
modified as per your comment.